.editorconfig

```diff
@@ -2,7 +2,7 @@ root = true
 
 [*]
 trim_trailing_whitespace = true
-insert_final_newline = true
+insert_final_newline = false
```

Comment:
controversial!
a stack of the other projects have it the way it was?
(While I got sold on it a few years back as being the right thing, but it still irks me to this day, and obviously some IDE / tool generated files don't respect it and hence cause null diffs all the time)

Reply:
In this one it's the public API surface area test, which depends on a diff. If I try to edit the "approved" side, the inserted newline causes a test failure. Irked me for too long now :-)

Reply:
https://github.com/serilog/serilog/pull/2028